### PR TITLE
Kanto CM docs

### PR DESCRIPTION
**Pull request by mikehaller**
Added documentation about Kanto Container Management MQTT messages flows, with two examples:
- Container Management initial response to cloud connectivity (list of containers and their details)
- Container Metrics: CPU and Memory for each container in a regular interval

### CODE REVIEW

File: content/en/docs/device-provisioning/self-update/self-update-tutorial.md [hunk 1]
```diff
@@ -34,6 +34,182 @@ This chapter describes the steps necessary to perform a local (without cloud) se
...
+      ```
+
+
+### Desired State
+
+1. External trigger to update via desired state on topic `selfupdate/desiredstate`:
+
+      ```json
+      {
+            "activityId": "random-uuid-as-string",
```
Comment (stlachev) [inline content/en/docs/device-provisioning/self-update/self-update-tutorial.md:94]: Is it like previous "activityId": "<uuid>",

File: content/en/docs/device-provisioning/self-update/self-update-tutorial.md [hunk 2]
```diff
@@ -34,6 +34,182 @@ This chapter describes the steps necessary to perform a local (without cloud) se
...
+                    ]
+                }
+      }
+      ```
+
+5. Response by Self Update Agent on topic `selfupdate/desiredstatefeedback` while performing the installation:
+
+      ```json
+      {
+            "activityId": "random-uuid-as-string",
```
Comment (stlachev) [inline content/en/docs/device-provisioning/self-update/self-update-tutorial.md:184]: same as previohs "activityId": "<uuid>",
probably I'm missing something

**Review by stlachev — APPROVED** [review]
LGTM

File: content/en/docs/device-provisioning/self-update/self-update-tutorial.md (hunk 1 above)
Comment (mikehaller) [reply content/en/docs/device-provisioning/self-update/self-update-tutorial.md]: Fixed

File: content/en/docs/device-provisioning/self-update/self-update-tutorial.md (hunk 2 above)
Comment (mikehaller) [reply content/en/docs/device-provisioning/self-update/self-update-tutorial.md]: Fixed

**Pull request merged**